src/Apps/Artist/ArtistApp.tsx

```diff
@@ -26,21 +37,52 @@ export interface ArtistAppProps {
 export const ArtistApp: React.FC<ArtistAppProps> = props => {
   const { artist, children } = props
 
+  const route = findCurrentRoute((props as any).match)
+
   return (
     <AppContainer>
       <ArtistMeta artist={artist} />
-      <Row>
-        <Col>
-          <ArtistHeader artist={artist} />
-        </Col>
-      </Row>
+      {route.displayNavigationTabs && (
```

Comment:
Why is the navigation conditionally rendered?

Reply:
Based on the designs, we either show the navigation tabs or a back-link.